src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -282,13 +294,9 @@ to the `<a>` tag. On Android, links with `target="_blank"` open in a
 A few other notes on `scope`:
 
 * If you don't include a `scope` in your manifest, then the default implied
-  `scope` is the directory that your web app manifest is served from.
-* The `scope` attribute can be a relative path (`../`), or any higher level
-  path (`/`) which would allow for an increase in coverage of navigations
-  in your web app.
+  `scope` is the start URL but with its filename, query, and fragment removed.
+* If `scope` attribute is a relative path, the manifest URL is used as a base URL.
 * The `start_url` must be in the scope.
-* The `start_url` is relative to the path defined in the `scope` attribute.
```

Comment:
I couldn't find a source for this in the specs so it may be wrong, hence removing it

Reply:
this is not for the general spec, but for PWA